src/RedirectConfig.js

```diff
@@ -36,7 +36,7 @@ class RedirectConfig extends SchemaDerivedConfig {
     });
   }
 
-  match(path) {
+  async match(path) {
```

Comment:
what would be an example of an async match behaviour?

Reply:
I'm working on Redirect configs that are spreadsheet based, and fetching the spreadsheet would be async.

Reply:
See #327